### PR TITLE
export dist

### PR DESCRIPTION
**Pull request by jonborchardt**

### CODE REVIEW

File: tsconfig.bundle.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "module": "amd",
```
Comment (jonborchardt) [inline tsconfig.bundle.json:4]: is amd right?

File: webpack.config.js [hunk 2]
```diff
@@ -3,10 +3,6 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 const config = {
     entry: './app/index.tsx',
-    output: {
```
Comment (jonborchardt) [inline webpack.config.js:6]: i dont think these are used anymore

File: tsconfig.bundle.json (hunk 1 above)
Comment (codeviking) [reply tsconfig.bundle.json:4]: So, AMD modules use RequireJS: https://requirejs.org/docs/whyamd.html

It depends on what CodePen wants. The other module type that'd be applicable would be `"system"`, which uses SystemJS: https://github.com/systemjs/systemjs

I like SystemJS more, so if they support either, I'd use that -- but it's a subjective opinion so use what works and you like :).

File: webpack.config.js (hunk 2 above)
Comment (codeviking) [reply webpack.config.js:6]: Great!

Comment (codeviking) [reply webpack.config.js:6]: As long as `yarn start` still works and you can see the site at http://localhost:8080 -- did you check that?

File: tsconfig.bundle.json (hunk 1 above)
Comment (jonborchardt) [reply tsconfig.bundle.json:4]: ill convert to `system`

File: webpack.config.js (hunk 2 above)
Comment (jonborchardt) [reply webpack.config.js:6]: yes, still works

**Pull request merged**